### PR TITLE
Align top nav and disclaimer with dashboard width [IGNORE FOR NOW]

### PR DESCRIPTION
**Pull request by thisisdano**
**Ignore in favor of dropping the sidebar**

This PR adjusts the padding and max-widths of the disclaimer and the top nav of the dashboard to align their left and right edges with the outside edges of the dashboard content. It adjusts the `$medium-screen` responsive width to better determine when the `disclaimer-stage` text should show. The end result is a cleaner visual presentation of the top of the page.

This affects only dashboard with a kludgey style hook: `.js-app`. I used this hook only because it was an existing class that is unique to the dashboard. We could apply a better hook, perhaps at the `body` level, but I didn't want to touch `cg-dashboard`. 

Can someone help me determine where to put this style hook and what it should be?

- - -

screen shot 2016-12-29 at 10 45 59 am

### CODE REVIEW

File: src/css/components/header.scss [hunk 1]
```diff
@@ -16,6 +16,15 @@
   padding-bottom: $grid-3;
 }
 
+// kludge for targeting dashboard
+.js-app {
```
Comment (msecret) [inline src/css/components/header.scss]: Rather then add more kludge, why don't we add another header class, like `.header-lg` or `header-full`.

File: src/css/override_components.scss [hunk 2]
```diff
@@ -45,6 +48,22 @@
   }
 }
 
+// kludge for targeting dashboard
+.js-app {
```
Comment (msecret) [inline src/css/override_components.scss]: Same as above, maybe `disclaimer-lg`, or wrap the disclaimer in the `header-lg` or whatever class.

**Review by msecret — CHANGES_REQUESTED** [review]

File: src/css/components/header.scss (hunk 1 above)
Comment (thisisdano) [reply src/css/components/header.scss]: Think I'll do this, wrap in `header-wide`

File: src/css/override_components.scss (hunk 2 above)
Comment (thisisdano) [reply src/css/override_components.scss]: Think I'll do this, wrap in `disclaimer-wide`

**Comment by thisisdano** [comment]
@msecret fixed the class hooks per your request. Now paired with https://github.com/18F/cg-dashboard/pull/884

**Pull request closed without merging**